### PR TITLE
style(planner-layout): fix styles for planner layout

### PR DESCRIPTION
**Pull request by SMahil**
### Mandatory
 - [ ] What does this PR do? 
This PR will 
- remove the unused CSS
- separate the multiple styles into separate ones

- [ ] What issue/task does this PR references?
https://openshift.io/openshiftio/Openshift_io/plan/detail/104
- [ ] Are the tests Included?
NA
___

- [ ] @mention(s) to expected reviewer(s) included
@nimishamukherjee

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @SMahil for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-ui/fabric8-planner/blob/26478d0a12d14f8c7b86e2a1d6d0498d3c1d3b9c/.ike-prow/test-keeper.yaml).

**Comment by SMahil** [comment]
[test]

**Comment by jarifibrahim** [comment]
[test]

**Comment by SMahil** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by vikram-raj** [comment]
screenshot from 2018-07-31 16-32-33

@SMahil Work item type dropdown is not visible in case of inline quick add.

File: src/app/components_ngrx/planner-list/planner-list.component.less [hunk 1]
```diff
@@ -36,16 +37,16 @@
...
-    height: 100%;
     background-color: @color-pf-white;
     &-wrap {
       position: relative;
       width: 100%;
-      // height: ~"calc(e('100% - 120px'))";
-      // to avoid overflown scroll height should be there
       overflow-y: auto;
+      .f8-planner-list-title {
+        max-width: ~"calc(e('100% - 20px'))";// lesshint-disable-line
```
Comment (vikram-raj) [inline src/app/components_ngrx/planner-list/planner-list.component.less]: make it as ```max-width: ~'calc(e("100% - 20px"))'``` then there is no need of ```// lesshint-disable-line``` comment.

Comment (vikram-raj) [reply src/app/components_ngrx/planner-list/planner-list.component.less]: It should work. See here https://github.com/fabric8-ui/fabric8-planner/blob/master/src/app/components_ngrx/planner-board-column/planner-board-column.component.less#L28

**Comment by SMahil** [comment]
[test]

**Comment by vikram-raj** [comment]
screenshot from 2018-08-09 21-57-16

State ```No Plan to Implement``` is overflow from the column.

**Comment by vikram-raj** [comment]
screenshot from 2018-08-09 22-05-27

When I expand tree then loader move upward.

File: src/app/components_ngrx/work-item-cell/work-item-cell.component.html [hunk 2]
```diff
@@ -42,7 +42,7 @@
 <div *ngIf="col === 'state'">
   <span class="padding-right-5 padding-top-4" almIcon [iconType]="row.state">
   </span>
-  <span>
+  <span style="white-space: normal;">
```
Comment (vikram-raj) [inline src/app/components_ngrx/work-item-cell/work-item-cell.component.html]: Why did you add this inline style?

Comment (SMahil) [reply src/app/components_ngrx/work-item-cell/work-item-cell.component.html]: considering it as temporary solution

Comment (vikram-raj) [reply src/app/components_ngrx/work-item-cell/work-item-cell.component.html]: Line height for this should be also normal otherwise it looks like this

screenshot from 2018-08-24 13-48-50

File: src/app/components_ngrx/work-item-cell/work-item-cell.component.less [hunk 3]
```diff
@@ -23,3 +23,10 @@
     margin: 0 !important;
   }
 }
+.f8-planner-cell__state {
+  display: flex;
+  &-text {
+    line-height: normal;
+    white-space: normal;
+  }
+}
```
Comment (vikram-raj) [inline src/app/components_ngrx/work-item-cell/work-item-cell.component.less]: After applying these properties all the other align vertically top. Because text is vertically aligned middle by line height. So, this should be 
```
.f8-planner-cell__state {
  display: table;
  &-text {
    line-height: normal;
    white-space: normal;
    display: table-cell;
    vertical-align: middle;
  }
```

File: src/app/components_ngrx/planner-list/ngx-datatables.less [hunk 4]
```diff
@@ -3,9 +3,18 @@
 
 .cell-class {
   overflow: visible !important;
-  padding-top: 3px !important;
-  padding-bottom: 4px !important;
-  line-height: 44px !important;
+  align-items: center;
+  height: 100%;
+  vertical-align: middle;
```
Comment (vikram-raj) [inline src/app/components_ngrx/planner-list/ngx-datatables.less]: I do not see any use of these three properties here. ```align-items: center; height: 100%; vertical-align: middle;```.

Comment (SMahil) [reply src/app/components_ngrx/planner-list/ngx-datatables.less]: cell items can be impacted if we don't set it to center and height 100% makes sure any item takes full height to keep everything vertically centered

File: src/app/components_ngrx/planner-list/ngx-datatables.less [hunk 5]
```diff
@@ -0,0 +1,101 @@
+@import (reference) '../../../assets/stylesheets/_base';
+/// ngx-datatables css
+
+.cell-class {
+  overflow: visible !important;
+  align-items: center;
+  height: 100%;
```
Comment (vikram-raj) [inline src/app/components_ngrx/planner-list/ngx-datatables.less]: > cell items can be impacted if we don't set it to center and height 100% makes sure any item takes full height to keep everything vertically centered

```align-items: center``` is only working with flexbox and ```cell-class``` is not a flexbox. And we are already setting cell height here https://github.com/fabric8-ui/fabric8-planner/blob/master/src/app/components_ngrx/planner-list/planner-list.component.html#L102.

**Comment by vikram-raj** [comment]
screenshot from 2018-08-31 12-13-37

When I expand tree then loader move down.

**Comment by SMahil** [comment]
[test]
